CHANGELOG.md

```diff
@@ -9,6 +9,13 @@
     - Not recommended: Use `core::executor::block_on(value)` to retrieve the `value`
 - `syscalls::yieldk_for` is no longer available
   - Yielding manually is discouraged as it conflicts with Rust's safety guarantees. If you need to wait for a condition, use `futures::wait_until` and `.await`.
+- `syscalls::yieldk` has become `unsafe` for the same reason
+- Commands are no longer `unsafe`
+- The low-level syscalls have been moved to `syscalls::raw`
+  - `syscalls::subscribe_ptr` becomes `syscalls::raw::subscribe`
+  - `syscalls::command1_insecure` becomes `syscalls::raw::command1`
```

Comment:
`command1_insecure` should keep the `_insecure` postfix. It was intentionally named that way because it leaks data to kernel capsules (i.e. it is very easy to cause a security vulnerability with it). I am okay with making it safe, but not with the renaming.

Reply:
@jrvanwhy Thanks for your feedback! I was expecting that the `_insecure` postfix had some intention. My suggestion was to move everything "unsafe* to the `raw` module which implies some kind of unsafety but I am aware that people could think differently about it.

Reply:
The code is back to `command1_insecure`